packaging/ubuntu-14.04/rules

```diff
@@ -130,6 +131,9 @@ override_dh_auto_build:
 	cd cmd && ( ./configure --prefix=/usr --libexecdir=/usr/lib/snapd $(VENDOR_ARGS))
 	$(MAKE) -C cmd all
 
+	# Generate the env snippet
+	$(MAKE) -C data/env all
```

Comment:
Could we have a single "$(MAKE) -C data all" here?

Reply:
14.04 wasn't making dbus. Is that a bug?

Reply:
Probably :/